### PR TITLE
(rest) Add Schema index and Aggregate

### PR DESCRIPTION
**Pull request by dplewis**
Adds documentation for schema index, aggregate and distinct queries

Pending: Parse-Server 3.0

### CODE REVIEW

**Comment by flovilmart** [comment]
it's been released part of 2.7, let's merge!

File: _includes/rest/schemas.md [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
 of an app to verify operations in a unit test, generate test data, generate test
 classes and then clean up after tests. The schema API can also be used to create
-custom views of your data. We use the schema API to display columns names and 
+custom views of your data. We use the schema API to display columns names and
 types in the databrowser.
 
 This API allows you to access the schemas of your app.
 Note: This API can be only accessed using the `master key`.
 
+* Starting with Parse-Server 3.0 Index support added.
```
Comment (dplewis) [inline _includes/rest/schemas.md:12]: Should change this to 2.7.0

File: _includes/rest/queries.md [hunk 2]
```diff
@@ -941,3 +941,161 @@ print result
...
+       "X-Parse-Master-Key": "${MASTER_KEY}"
+       "X-Parse-REST-API-Key": "${REST_API_KEY}"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+</code></pre>
+
+## Aggregate Queries
+
+* Starting with Parse-Server 3.0 (MasterKey Required)
```
Comment (dplewis) [inline _includes/rest/queries.md:1003]: 2.7.0

**Comment by flovilmart** [comment]
Yep that's fixed on master.

**Pull request merged**